packages/nc-gui-v2/plugins/api.ts

```diff
@@ -21,7 +21,7 @@ function addAxiosInterceptors(api: Api<any>, app: { $state: GlobalState }) {
   api.instance.interceptors.request.use((config) => {
     config.headers['xc-gui'] = 'true'
 
-    if (app.$state.token.value) config.headers['xc-auth'] = app.$state.token.value
+    if (app.$state?.token.value) config.headers['xc-auth'] = app.$state.token.value
```

Comment:
@bcakmakoglu Yes it is erroring out. But shouldn't state be populated? Or is it because this api call is happening in a plugin so state is not set by then?

![image](https://user-images.githubusercontent.com/35809690/181717293-0097ce19-6006-4225-87f5-4567fc9b69db.png)